src/libs/actions/Report.js

```diff
@@ -1011,12 +1012,48 @@ function fetchAllReports(
 
             // Get all the chat reports if they have any, otherwise create one with concierge
             if (reportIDs.length > 0) {
-                return fetchChatReportsByIDs(reportIDs);
+                // return fetchChatReportsByIDs(reportIDs);
+                const simplifiedReports = {};
+                return API.GetChats({reportIDList: reportIDs})
```

Comment:
Hmmm this is still doing 2 API calls: 1 to Get chatList and one to GetChats. We could put all the logic in Get chatList, no?

Reply:
Yup, I mentioned that in the issue where I asked for the reviews ;)